Lib/threading.py

```diff
@@ -1398,6 +1417,10 @@ def _after_fork():
     new_active = {}
     current = current_thread()
     _main_thread = current
+
+    with _shutdown_locks_lock:
```

Comment:
We could have forked while this lock was held. You'll need to reset it above. There can be no other threads running in our new process yet.

Reply:
Oops. You're right. It's now fixed.